src/filters/compress/config.rs

```diff
@@ -49,6 +50,22 @@ impl Default for Mode {
     }
 }
 
+impl From<Mode> for ProtoMode {
```

Comment:
Not sure if I was clear in the above comment which was resolved.

I was suggesting that we should follow the same pattern of using the `proto` import alias as we have elsewhere in this PR for this file as well for consistency (and I like the pattern better).

So this should become:

```suggestion
impl From<Mode> for proto::Mode {
```

and so on and so forth for the other items in this file as well.

Reply:
The problem with doing it for these types, is that they're embedded within the `compress` message namespace, so it would be `proto::compress::Mode::Snappy` for example rather than `proto::Mode::Snappy`. I could move the types outside the message in their Protobuf definition to fix that, but I wanted to make sure separately that that's what we want to do here separately.